sceptre/cli/list.py

```diff
@@ -78,11 +93,18 @@ def list_change_sets(ctx, path):
     List change sets for stack.
 
     """
-    stack, _ = get_stack_or_stack_group(ctx, path)
+    context = SceptreContext(
+                command_path=path,
+                project_path=ctx.obj.get("project_path", None),
+                user_variables=ctx.obj.get("user_variables", {}),
+                options=ctx.obj.get("options", {})
```

Comment:
Should we be passing the `output_format=ctx.obj["output_format"]` here?

Reply:
agreed, resolved.